yarn.lock

```diff
@@ -1677,6 +1679,11 @@ character-parser@^2.2.0:
   dependencies:
     is-regex "^1.0.3"
 
+charcodes@^0.2.0:
```

Comment:
Introduced that dependency but goal is to remove https://github.com/xtuc/acorn-import-assertions/issues/9.

Reply:
``` js
import { leftCurlyBrace, space } from "charcodes";
```

can be replaced with

``` js
const leftCurlyBrace = "{".charCodeAt(0);
const space = " ".charCodeAt(0);
```

No need for a package or babel transfrom...